service/src/main/resources/application.yml

```diff
@@ -7,7 +7,6 @@ drshub:
 
   compactIdHosts:
     # it is important for case-insensitive matching that the keys here are all lower case
-    "dg.4503": gen3.biodatacatalyst.nhlbi.nih.gov
```

Comment:
The "prod" and "!prod" definitions for "dg.4503" and "dg.712c" later in this file look good and make sense.

I am less clear about when the definitions in this section would come into play.
Depending on when/how the values in this section are used, my general sense is that either:
* Both "dg.4503" and "dg.712c" would remain here. 
* or, both "dg.4503" and "dg.712c" would be removed here
Please advise. :)

Reply:
I had a look at the docs, the default case will appear in all envs unless explicitly overwritten, so I removed both of the env-specific CIBs from here, so each should only populate in the correct env now. Thanks for catching that!

Reply:
For the same reason, these can(/should) also be removed from the common block, as these all correctly appear in the `prod` and `!prod` blocks below:
```
    "dg.anv0": staging.theanvil.io
    "drs.anv0": jade.datarepo-dev.broadinstitute.org
    "dg.4dfc": nci-crdc-staging.datacommons.io
    "dg.f82a1a": gen3staging.kidsfirstdrc.org
  ```